waspc/data/Generator/templates/sdk/wasp/client/webSocket/WebSocketProvider.tsx

```diff
@@ -1,5 +1,5 @@
 {{={= =}=}}
-import { createContext, useState, useEffect, Context } from 'react'
+import React, { createContext, useState, useEffect, Context } from 'react'
```

Comment:
The default export in React is soft-deprecated. We can just import what we want.

```suggestion
import { createContext, useState, useEffect, Context, ReactNode } from 'react'
```